course_content/notebooks/iris_intro.ipynb

```diff
@@ -630,37 +589,29 @@
     "collapsed": true
    },
    "outputs": [],
-   "source": [
-    "c1, = iris.load(fname, 'surface_altitude')\n",
-    "c2 = iris.load_cube(fname, 'surface_altitude')\n",
-    "c3, = iris.load_cubes(fname, 'surface_altitude')\n",
-    "c1 == c2 == c3"
-   ]
+   "source": []
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In general, it is a good idea to make use of the ``load_cube``/``load_cubes`` functions rather than the generic ``load`` function in non-exploratory code. Doing so makes your code more resilient to changes in the data source, often results in more readable/maintainable code, and in combination with well defined constraints, often leads to improve load performance."
+    "4\\. Get hold of the `latitude` coordinate on the `sea_water_potential_temperature` cube. Identify whether this coordinate has bounds. Print the minimum and maximum latitude points in the cube."
    ]
   },
   {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "The load functions all accept a list of filenames to load, and any of the filenames can be \"glob\" patterns (http://docs.python.org/2/library/glob.html)."
-   ]
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Exercise 2 (continued)\n",
-    "\n",
-    "Read in the files found at **``iris.sample_data_path('GloSea4', 'ensemble_010.pp')``** and\n",
-    "**``iris.sample_data_path('GloSea4', 'ensemble_011.pp')``** using a single load call. Do this by:\n",
-    "\n",
-    "1\\. providing a list of the two filenames."
+    "5\\. Go to the Iris reference documentation for ``iris.save``. What fileformats can Iris currently save to? What keywords are accepted to ``iris.save`` when saving a PP file?"
```

Comment:
All of these exercises are much better.  I heartily approve.

Reply:
🏅 🌟 ⭐️ 💯